source/Tutorials/Advanced/Discovery-Server/Discovery-Server.rst

```diff
@@ -121,9 +131,22 @@ Launch talker node
 
 Open a new terminal and set the ``ROS_DISCOVERY_SERVER`` environment variable as before so that the node starts a discovery client.
 
+.. tabs::
+
+    .. group-tab:: Linux
+
+        .. code-block:: console
+
+            export ROS_DISCOVERY_SERVER=127.0.0.1:11811
+
+   .. group-tab:: Windows
+
+        .. code-block:: console
+
+            set export ROS_DISCOVERY_SERVER=127.0.0.1:11811
```

Comment:
I think this should be set ROS_DISCOVERY_SERVER=127.0.0.1:11811

Reply:
```suggestion
            set ROS_DISCOVERY_SERVER=127.0.0.1:11811
```